scipy/stats/stats.py

```diff
@@ -6545,12 +6545,17 @@ def chisquare(f_obs, f_exp=None, ddof=0, axis=0):
     See Also
     --------
     scipy.stats.power_divergence
+    fisher_exact : A more powerful alternative to the chisquare test if any
+        of the frequencies are less than 5.
 
     Notes
     -----
     This test is invalid when the observed or expected frequencies in each
     category are too small.  A typical rule is that all of the observed
-    and expected frequencies should be at least 5.
+    and expected frequencies should be at least 5. If one or more frequencies
+    are less than 5, Fisher's Exact Test can be used with greater statistical
+    power. The total number of samples must be greater than 13, otherwise
+    a table-based method of obtaining p-values is recommended.
```

Comment:
I don't think we enforce a minimum count of 13. So, instead of writing, "The total number of samples *must* be greater than 13", we should write something like:

```suggestion
    power. According to [3]_, the total number of samples is recommended
    to be greater than 13, otherwise a table-based method of obtaining
    p-values is recommended.
```